sys/include/net/gnrc/sixlowpan/config.h

```diff
@@ -81,6 +81,22 @@ extern "C" {
 #define GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US (3U * US_PER_SEC)
 #endif
 
+/**
+ * @brief   Keep all but oldest fragment when reassembly buffer is full
+ *
+ * @note    Only applicable with
+ *          [gnrc_sixlowpan_frag_rb](@ref net_gnrc_sixlowpan_frag_rb) module
+ *
+ * When not set, it will cause the reassembly buffer to
+ * override the oldest entry no matter what. When set, only the oldest
+ * entry that is older than @ref GNRC_SIXLOWPAN_FRAG_RBUF_TIMEOUT_US will be
+ * overwritten (they will still timeout normally if reassembly buffer is not
+ * full).
+ */
+#ifndef GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE
```

Comment:
As we are doing this change to model this parameter like a Kconfig boolean (when set to 'n' the macro is not defined), something like this has a closer behaviour: 
```C
#ifdef DOXYGEN
#define GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE
#endif
```
And for the translation:
```C
#ifndef GNRC_SIXLOWPAN_FRAG_RBUF_AGGRESSIVE_OVERRIDE
#ifdef IS_ACTIVE(GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE)
#define GNRC_SIXLOWPAN_FRAG_RBUF_AGGRESSIVE_OVERRIDE   (0)
#else /* GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE */
#define GNRC_SIXLOWPAN_FRAG_RBUF_AGGRESSIVE_OVERRIDE    (1)
#endif /* GNRC_SIXLOWPAN_FRAG_RBUF_DO_NOT_OVERRIDE */
#endif /* GNRC_SIXLOWPAN_FRAG_RBUF_AGGRESSIVE_OVERRIDE */
```